GVFS/GVFS.UnitTests/Platform.Mac/MacDaemonManagerTests.cs

```diff
@@ -0,0 +1,41 @@
+using GVFS.Common;
+using GVFS.Platform.Mac;
+using GVFS.Tests.Should;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace GVFS.UnitTests.Platform.Mac
+{
+    [TestFixture]
+    public class MacServiceProcessTests

```

Comment:
Can we add any tests for `IsServiceInstalledAndRunning`?

Reply:
Noted for followup